Qube TechRadar.csv

```diff
@@ -2,4 +2,5 @@ name,ring,quadrant,isNew,description
 Go,Adopt,languages-and-frameworks,FALSE,"Go is an open source programming language that makes it easy to build simple, reliable, and efficient software."
 Visual Studio Code,Trial,Tools,FALSE,"Visual Studio Code is a code editor redefined and optimized for building and debugging modern web and cloud applications."
 AWS DevOps,Trial,Platforms,FALSE,"AWS DevOps provides services to simplify provisioning and managing infrastructure, deploying application code, automating software release processes, and monitoring your application and infrastructure performance."
-Microservices,Adopt,Techniques,FALSE,"Microservices arranges an application as a collection of loosely coupled services."
\ No newline at end of file
+Microservices,Adopt,Techniques,FALSE,"Microservices arranges an application as a collection of loosely coupled services."
+Gstreamer,Adopt,framework,FALSE,"Gstreamer is an open source multimedia framework which allows a wide range of media content handling from simple Ogg/Vorbis playback, audio/video streaming to complex audio (mixing) and video (non-linear editing) processing."
```

Comment:
I believe you'll have to use the exact words `languages-and-frameworks` for your entry to fall under the right quadrant.

Also, can we add a new line character at the end so that the next person who is editing it will not see 2 lines of change like you are seeing right now.

```suggestion
Gstreamer,Adopt,languages-and-frameworks,FALSE,"Gstreamer is an open source multimedia framework which allows a wide range of media content handling from simple Ogg/Vorbis playback, audio/video streaming to complex audio (mixing) and video (non-linear editing) processing.
"
```

Reply:
Done.